libkbfs/md_journal.go

```diff
@@ -479,15 +490,19 @@ func (j *mdJournal) convertToBranch(
 		}
 		mdsToRemove = append(mdsToRemove, newID)
 
-		err = tempJournal.append(brmd.RevisionNumber(), newID)
+		// No need to try and preserve unknown fields from the
```

Comment:
I thought we'd need to try and preserve unknown fields when rewriting the journal, but now I'm not so sure. What do you think?